src/iris_api/api.py

```diff
@@ -1875,6 +1875,14 @@ def validate_app(app):
             for mid in mid_lst:
                 self.create_response(mid, source, content)
             is_batch = True
+        elif isinstance(msg_id, list) and content == 'claim_all':
+            # Claim all functionality.
+            if not msg_id:
+                raise HTTPBadRequest('Invalid message id', 'No incidents to claim')
+            app = get_app_from_msg_id(session, msg_id[0])
```

Comment:
can/should we assume that all of the claimed incident messages are from one application?